docs/catalogsource.md

```diff
@@ -0,0 +1,43 @@
+# Create a CatalogSource to test manifest changes
+
+## Create manifests
+```bash
+make manifests
+```
+
+## Create registry
+
+### Create registry Dockerfile
+```bash
+cat <<EOF > LocalRegistryDockerfile
+FROM quay.io/openshift/origin-operator-registry:latest
+
+COPY manifests manifests
+RUN initializer --permissive
+
+CMD ["registry-server", "-t", "/tmp/terminate.log"]
+EOF
+```
+
+### Build and push Dockerfile
+
+```bash
+docker build -f ./LocalRegistryDockerfile --tag quay.io/<your_repo>/managed-upgrade-operator-registry:<tag> .
```

Comment:
should we suggest to use `podman` or `imagebuilder` instead of docker?

Reply:
I'm just replicating what the app-sre scripts do. I think sticking to that is of more benefit since we will encounter any issue the same as they will do